rules/rules-reviewed/eap8/eap7/hibernate6.windup.xml

```diff
@@ -0,0 +1,530 @@
+                    <javaclass references="org.hibernate.loader.plan.{*}">
+                        <location>IMPORT</location>
+                    </javaclass>
+                </or>
+            </when>
+            <perform>
+                <hint title="The contents of the loader.entity and loader.plan packages have been removed" category-id="mandatory" effort="1">
+                    <message>The contents of `loader.entity` and `loader.plan` were removed</message>
+                    <link href="https://github.com/hibernate/hibernate-orm/blob/6.0/migration-guide.adoc#restructuring-of-orghibernateloader" title="Hibernate ORM 6 migration guide - Restructuring of org.hibernate.loader"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="hibernate6-00140">
+            <when>
+                <javaclass references="org.hibernate.sql.ordering.{*}">
+                    <location>IMPORT</location>
+                </javaclass>
+            </when>
+            <perform>
+                <hint title="The contents of sql.ordering have moved" category-id="mandatory" effort="1">
+                    <message>The contents of `sql.ordering` were adapted and moved to `metamodel.mapping.ordering.ast`.</message>
+                    <link href="https://github.com/hibernate/hibernate-orm/blob/6.0/migration-guide.adoc#restructuring-of-the-sql-package" title="Hibernate ORM 6 migration guide - Restructuring of the SQL package"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="hibernate6-00150">
+            <when>
+                <javaclass references="org.hibernate.sql.{class-name}"/>
+            </when>
+            <perform>
+                <hint title="Some classes of the sql package that were previously used for building SQL were removed" category-id="mandatory" effort="1">
+                    <message>Classes of the sql package that were previously used for building SQL, but aren’t needed anymore, were removed. The SQL generation is now fully handled through the `SqlAstTranslator` which a `Dialect` exposes a factory for.</message>
+                    <link href="https://github.com/hibernate/hibernate-orm/blob/6.0/migration-guide.adoc#restructuring-of-the-sql-package" title="Hibernate ORM 6 migration guide - Restructuring of the SQL package"/>
+                </hint>
+            </perform>
+            <where param="class-name">
+                <matches pattern="(ANSICaseFragment|ANSIJoinFragment|CacheJoinFragment|CaseFragment|ConditionFragment|DecodeCaseFragment|DerbyCaseFragment|DisjunctionFragment|HSQLCaseFragment|InsertSelect|JoinFragment|JoinType|MckoiCaseFragment|OracleJoinFragment|QueryJoinFragment|QuerySelect|Select|SelectExpression|SelectFragment|SelectValues|Sybase11JoinFragment)"/>
+            </where>
+        </rule>
+        <rule id="hibernate6-00160">
+            <when>
+                <xmlfile matches="//*/property[count(column) > 1]" in="hbm.xml"/>
```

Comment:
Usually `xmlfile` conditions need the relevant `namespace` inner tag to work properly or  `system-id` (or `public-id`) attribute [1].

I think it should understood what is expected to have in the source migration to properly enhance the rule and the corresponding test file.

[1] https://github.com/windup/windup-rulesets/blob/master/rules/rules-reviewed/hibernate/hibernate-xml.windup.xml#L75